packages/commerce-sdk-react/setup-jest.js

```diff
@@ -39,6 +39,9 @@ Object.defineProperty(window, 'localStorage', {
     value: localStorageMock
 })
 
-global.beforeEach(() => {
+global.afterEach(() => {
+    nock.cleanAll()
+})
+global.afterAll(() => {
     nock.restore()
```

Comment:
We still need to call `restore` to solve the current problem of nock leaking into test suites that do not use nock.